testing/server.go

```diff
@@ -28,12 +29,25 @@ import (
 // Callers are responsible of closing the server by calling Close().
 func OpenServer(c *gc.C, session *mgo.Session, params charmstore.ServerParams) *Server {
 	db := session.DB("charmstore-testing")
+	if params.AuthUsername == "" {
+		params.AuthUsername = "charmstore-testing-user"
```

Comment:
We should probably declare these as constants and document that
they're used like this.